scio-core/src/main/scala/com/spotify/scio/ScioResult.scala

```diff
@@ -176,4 +178,8 @@ class ScioResult private[scio] (val internal: PipelineResult,
     aggregators.getOrElse(acc.name, Nil)
```

Comment:
+1 makes sense to lock it down here instead. What do you think of this in general though (see my other comment), is it better to stub out the impl rather than prevent? So that it can still be used when actually running the template job.